### ISSUE TITLE
problems installing version 0.9.0 with pg 9.6 (ubuntu 16.04)

### ISSUE THREAD

**Issue by paulovieira** [timescale/timescaledb#457]
I have PG 9.6 running on Ubuntu 16.04:

```
pvieira=# show server_version;
 server_version 
----------------
 9.6.8
(1 row)
```

I installed timescale following the apt instructions: http://docs.timescale.com/v0.9/getting-started/installation/linux/installation-apt

When I restart the server I have the following error:

```
postgres@pvieira-hp:~$ /usr/lib/postgresql/9.6/bin/pg_ctl -D /var/lib/postgresql/9.6/main restart                                                                                               
                                                                                                                                                                               
server starting                                                                                                                                                                                
postgres@pvieira-hp:~$ 2018-03-06 17:23:06.288 WET [12288] FATAL:  incompatible library "/usr/lib/postgresql/9.6/lib/timescaledb.so": version mismatch                                          
2018-03-06 17:23:06.288 WET [12288] DETAIL:  Server is version 9.6, library is version 10.0.
2018-03-06 17:23:06.288 WET [12288] LOG:  database system is shut down
```
This seems to be similar to the issue #394.

Thanks in advance for any help.

NOTE: I'm absolutely sure I installed timescale with
```
sudo apt install timescaledb-postgresql-9.6
```
This can be verified with:
```
pvieira@pvieira-hp:~$ dpkg -L timescaledb-postgresql-9.6
/.
/usr
/usr/lib
/usr/lib/postgresql
/usr/lib/postgresql/9.6
/usr/lib/postgresql/9.6/lib
/usr/lib/postgresql/9.6/lib/timescaledb.so
/usr/lib/postgresql/9.6/lib/timescaledb-0.9.0.so
/usr/share
/usr/share/doc
/usr/share/doc/timescaledb-postgresql-9.6
/usr/share/doc/timescaledb-postgresql-9.6/copyright
/usr/share/doc/timescaledb-postgresql-9.6/changelog.gz
/usr/share/postgresql
/usr/share/postgresql/9.6
/usr/share/postgresql/9.6/extension
/usr/share/postgresql/9.6/extension/timescaledb--0.8.0--0.9.0.sql
/usr/share/postgresql/9.6/extension/timescaledb.control
/usr/share/postgresql/9.6/extension/timescaledb--0.7.1--0.9.0.sql
/usr/share/postgresql/9.6/extension/timescaledb--0.7.0--0.9.0.sql
/usr/share/postgresql/9.6/extension/timescaledb--0.9.0.sql
/usr/share/postgresql/9.6/extension/timescaledb--0.6.1--0.9.0.sql
/usr/share/postgresql/9.6/extension/timescaledb--0.5.0--0.9.0.sql
/usr/share/postgresql/9.6/extension/timescaledb--0.6.0--0.9.0.sql
/usr/share/postgresql/9.6/extension/timescaledb--0.3.0--0.9.0.sql
/usr/share/postgresql/9.6/extension/timescaledb--0.2.0--0.9.0.sql
/usr/share/postgresql/9.6/extension/timescaledb--0.4.1--0.9.0.sql
/usr/share/postgresql/9.6/extension/timescaledb--0.1.0--0.9.0.sql
/usr/share/postgresql/9.6/extension/timescaledb--0.4.2--0.9.0.sql
/usr/share/postgresql/9.6/extension/timescaledb--0.4.0--0.9.0.sql
pvieira@pvieira-hp:~$ 
```
On the other hand
```
pvieira@pvieira-hp:~$ dpkg -L timescaledb-postgresql-10
dpkg-query: package 'timescaledb-postgresql-10' is not installed
Use dpkg --info (= dpkg-deb --info) to examine archive files,
and dpkg --contents (= dpkg-deb --contents) to list their contents.
pvieira@pvieira-hp:~$ 
```

**Comment by RobAtticus** [comment]
Ooph, that's annoying, I believe you're right. I will get on making updated builds and push ASAP.

**Comment by paulovieira** [comment]
On the hand, the ` timescaledb-postgresql-10` also seems to be have its own problems with PG 10 (also on ubuntu 16.04):

```
pvieira=# show server_version;
          server_version          
----------------------------------
 10.3 (Ubuntu 10.3-1.pgdg16.04+1)
(1 row)

pvieira=# 
```

The installation fails with:
```
pvieira@pvieira-hp:~$ sudo apt install timescaledb-postgresql-10
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 timescaledb-postgresql-10 : Depends: postgresql-10 (< 10.3) but 10.3-1.pgdg16.04+1 is to be installed
E: Unable to correct problems, you have held broken packages.
pvieira@pvieira-hp:~$ 
```

**Comment by RobAtticus** [comment]
Big thanks for reporting these issues, we should have rebuilds of all packages soon. Very sorry for the inconvenience!

**Comment by RobAtticus** [comment]
Thanks again @paulovieira for reporting these issues. I have rebuilt and published new images for 9.6 for Ubuntu that should now work.

I have rebuilt PG 10 images that will allow using 10.3, those will be published shortly (within the hour).

Please test them out and let me know if you find any other issues.

**Comment by paulovieira** [comment]
The new .deb packages are now working well. Thanks for quick action to fix this! Impressive support from TSDB!
